src/main/java/io/cdap/e2e/pages/locators/CdfStudioLocators.java

```diff
@@ -148,7 +148,12 @@ public static WebElement locateSourceEndpointInCanvas(String sourcePluginName) {
 
   public static WebElement locateSinkNodeInCanvas(String sinkPluginName) {
     return SeleniumDriver.getDriver().findElement(
-      By.xpath("//*[contains(@data-cy,'plugin-node-" + sink + "') and @data-type='batchsink']"));
+      By.xpath("//*[contains(@data-cy,'plugin-node-" + sinkPluginName + "') and @data-type='batchsink']"));
```

Comment:
wonder how it was working earlier?
@sawantpritam are we using this method?

Reply:
It gave me an error when tried using it, that's how I found the bug.

Reply:
Right, I saw this but I wanted to stay away from using plugin title. I can remove my helper here and use it as a local helper in UI repo.

Reply:
The more I look at it, I think these are better suited for our (UI) own use. I can easily utilize some of the methods in `ElementHelper` class to get what I want. This PR will just be limited to that tiny bug fix.